packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3019,6 +3030,10 @@ class InputDecoration {
   /// or (b) the input has the focus.
   final String? hintText;
 
+  /// The widget to use inplace of the [hintText]. Either [hintText] or
```

Comment:
inplace => in place

Reply:
Also, the second sentence should be its own paragraph. From the [style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#dartdoc-specific-requirements):

"The first paragraph of any dartdoc section must be a short self-contained sentence that explains the purpose and meaning of the item being documented. "

Reply:
Updated